example-app/package.json

```diff
@@ -18,14 +18,15 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
-    "@admin-bro/express": "^3.0.0-beta.3",
-    "@admin-bro/mongoose": "^1.0.0-beta.3",
-    "@admin-bro/nestjs": "^1.0.0-beta.2",
+    "@admin-bro/express": "^3.0.1",
+    "@admin-bro/mongoose": "^1.1.0",
     "@nestjs/common": "^6.7.2",
     "@nestjs/core": "^6.7.2",
     "@nestjs/mongoose": "^7.0.2",
     "@nestjs/platform-express": "^6.7.2",
-    "admin-bro": "^3.0.0-beta.12",
+    "admin-bro": "^3.3.1",
```

Comment:
shouldn't all of these dependencies be as `peerDependencies` and `devDependencies`?
I think that `admin-bro-nestjs` shouldn't really install any dependencies for itself as every user would install them in their projects

Reply:
It's for example-app. Honestly I am not sure if this package.json is even resolved when installing library.